_topic_map.yml

```diff
@@ -328,6 +331,9 @@ Topics:
   File: create-images
 - Name: Managing imagestreams
   File: image-streams-manage
+- Name: Image controller configuration resources
```

Comment:
Is controller correct here?

Reply:
Also, should this move to the Registry bucket?

Reply:
no i think images is the right section for it.

Reply:
> Is controller correct here?

no, it's just "image configuration resources" i guess.  it controls the configuration of how images get consumed on the cluster, primarily.